.circleci/config.yml

```diff
@@ -154,6 +154,10 @@ references:
     run:
       name: Destroy Instances
       command: |
+        export PSWD=$(jq .serverLogsZipPassword < $(cat secret_conf_path) | cut -d \" -f 2)
```

Comment:
we are downloading the logs from the machines in the `destroy_instances.py`.
![image](https://user-images.githubusercontent.com/30797606/88105179-30b1e600-cbac-11ea-8d41-6aaef142488e.png)